Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -13,342 +16,411 @@
+	readable_output = f' ## Get feedback returned results: {response}'
 
-    return CommandResults(
-        readable_output=readable_output,
-        outputs_prefix='Arcanna.FeedbackField',
-        outputs=response
-    )
+	return CommandResults(
+		readable_output=readable_output,
+		outputs_prefix='Arcanna.FeedbackField',
+		outputs=response
+	)
 
 
 def set_default_job_id(client: Client, args: Dict[str, Any]) -> CommandResults:
-    job_id = args.get("job_id")
-    client.set_default_job_id(job_id)
-    return get_default_job_id(client)
+	job_id = args.get("job_id")
+	client.set_default_job_id(job_id)
+	return get_default_job_id(client)
 
 
 ''' MAIN FUNCTION '''
 
 
-def send_event_feedback(client: Client, feature_mapping_field: str, args: Dict[str, Any]) -> CommandResults:
-    job_id = args.get("job_id", None)
-    if not job_id:
-        job_id = client.get_default_job_id()
-    event_id = args.get("event_id")
-    mappings = parse_mappings(feature_mapping_field)
-
-    username = args.get("username")
-    label = args.get("label")
-    closing_notes = args.get("closing_notes", "")
-    indicators = args.get("indicators", None)
-    arcanna_label = mappings.get(label, None)
-    if arcanna_label is None:
-        raise Exception(f"Error in arcanna-send-feedback.Wrong label={label}")
-
-    response = client.send_feedback(job_id, event_id, username, arcanna_label, closing_notes, indicators)
-    readable_output = f' ## Arcanna send event feedback results: {response}'
-
-    return CommandResults(
-        readable_output=readable_output,
-        outputs_prefix='Arcanna.Event',
-        outputs_key_field='feedback_status',
-        outputs=response
-    )
-
-
-def send_bulk_events(client: Client, feature_mapping_field: str, args: Dict[str, Any]) -> CommandResults:
-    job_id = args.get("job_id")
-    events = argToList(args.get("events"))
-    mappings = parse_mappings(feature_mapping_field)
-    mapped_events = []
-    for event in events:
-        closing_status = event.get("closingReason")
-        closing_notes = event.get("closeNotes")
-        closing_user = event.get("closeUser")
-        arcanna_label = mappings.get(closing_status)
-        title = event.get("name")
-        severity = event.get("severity")
-
-        body = client.map_to_arcanna_raw_event(job_id, event, severity, title)
-        body["label"] = client.map_to_arcanna_label(arcanna_label, closing_notes, closing_user)
-
-        mapped_events.append(body)
-
-    response = client.send_bulk(job_id, mapped_events)
-    readable_output = f' ## Arcanna send bulk results: {response}'
-
-    return CommandResults(
-        readable_output=readable_output,
-        outputs_prefix='Arcanna.Bulk',
-        outputs_key_field='status',
-        outputs=response
-    )
-
+def send_event_feedback(client: Client, close_reasons: list, args: Dict[str, Any]) -> CommandResults:
+	job_id = args.get("job_id", None)
+	if not job_id:
+		job_id = client.get_default_job_id()
```

Comment:
Why do you need this? 
Why not pass the param of default_job_id to the function directly, and do something like that
job_id = args.get("job_id", default_job_id)

Reply:
applied suggestion